### ISSUE TITLE
[BUG] 1.4.x bug `s6-rc-compile: fatal`

### ISSUE THREAD

**Issue by nickv2002** [xthursdayx/yacreaderlibrary-server-docker#35]
I'm having an issue with the `latest` image (`sha256:9500a2ab240e174346ca1f0a8a2edd1357d24f61f39726d16609bd31f3cdc30f` or `sha256:8f559d28e4fdd415cd744fa421934062805a6586adfe6f89ee080f22a0815304`)

The YACReader service doesn't start and the Docker logs show the following:

> s6-rc-compile: fatal: during dependency resolution for service init-adduser: undefined service name init-script-check
> s6-rc: fatal: unable to take locks: No such file or directory
> s6-linux-init-shutdownd: warning: /run/s6/basedir/scripts/rc.shutdown exited 111
> s6-rc-compile: fatal: during dependency resolution for service init-adduser: undefined service name init-script-chec

For now I've just changed my server to use `xthursdayx/yacreaderlibrary-server-docker:v1.3.1` and that gets everything back to working order.  Thanks!

**Comment by sentience** [comment]
Same issue here with `9500a2ab240e`:
```
❯ sudo docker logs YACReaderLibraryServer
s6-rc-compile: fatal: during dependency resolution for service init-adduser: undefined service name init-script-check
```

**Comment by xthursdayx** [comment]
The latest tag versions of the p7zip and unarr images, which fix the bug you're both having, are the following:

[latest (sha256:111502a62b37e6bd6b286f0c3feb1b152f7c100edf216b6f7cf4292d36b2dde2)](https://hub.docker.com/layers/xthursdayx/yacreaderlibrary-server-docker/latest/images/sha256-111502a62b37e6bd6b286f0c3feb1b152f7c100edf216b6f7cf4292d36b2dde2?context=explore)
[v1.4.6 (sha256:111502a62b37e6bd6b286f0c3feb1b152f7c100edf216b6f7cf4292d36b2dde2)](https://hub.docker.com/layers/xthursdayx/yacreaderlibrary-server-docker/v1.4.6/images/sha256-111502a62b37e6bd6b286f0c3feb1b152f7c100edf216b6f7cf4292d36b2dde2?context=explore)
[p7zip (sha256:6d778ccd3a376934e64ed012e6732d80bc70802be902b7bd58683c89ad3f422f)](https://hub.docker.com/layers/xthursdayx/yacreaderlibrary-server-docker/p7zip/images/sha256-6d778ccd3a376934e64ed012e6732d80bc70802be902b7bd58683c89ad3f422f?context=explore)
and
[unarr (sha256:111502a62b37e6bd6b286f0c3feb1b152f7c100edf216b6f7cf4292d36b2dde2)](https://hub.docker.com/layers/xthursdayx/yacreaderlibrary-server-docker/unarr/images/sha256-111502a62b37e6bd6b286f0c3feb1b152f7c100edf216b6f7cf4292d36b2dde2?context=explore)

Based on the digest hashes you listed, you're not actually pulling the latest `latest` image. Please try removing your existing container and the docker image and pull the image again with the tag of your choosing.

**Comment by xthursdayx** [comment]
Apologies, a previous fix I'd released for this issue was only pushed to the `p7zip` tagged image, which I was using for testing, not the `latest`/`unarr` tags. Release [v1.4.7](https://github.com/xthursdayx/yacreaderlibrary-server-docker/releases/tag/v1.4.7) fixes this issue. Please pull the new image and report back if you're still having any issues. Cheers.

**Comment by nickv2002** [comment]
Thanks for the quick fix: pulling the 1.4.7 tag works for me. 👍
